src/components/core/AppSideBar.vue

```diff
@@ -161,25 +145,8 @@ export default {
   methods: {
     logout() {
       this.hideNav()
-      EventBus.$emit('events:click-logout-button', this)
       this.$store.dispatch('auth/LOGOUT_USER')
     },
-
-    // @TODO: instead of requesting these independently of
-    //  src/views/TransferListView.vue, the list of transfers should really be
-    //  retrieved & cached in vuex (or Resource pattern)
-    //
-    // @NOTE: this is also not responsive when a user ignores a transfer (until
```

Comment:
It seems like this `@NOTE` still applies - if I ignore a transfer it should update the badge.

Reply:
Thanks for pointing this out, fixing